### ISSUE TITLE
guess data type for all columns

### ISSUE THREAD

**Issue by Tagar** [fitzscott/AirQuality#7]
https://github.com/fitzscott/AirQuality/wiki/Guess-data-types

Would it be possible to guess data types for all columns in an exesting Hive table?

**Comment by fitzscott** [comment]
Yes, the code should work fine with a Hive table, too, though it works on files.  You'd end up with an alternative definition of your Hive table.
